db/queries/comments/queries.py

```diff
@@ -0,0 +1,69 @@
+"""Queries which are performed on the `scores` table.
+Joins allowed.
+"""
+
+import json
+from typing import Dict
+from typing import List
+from db import db
+from db.models.comment.comments import Comment
+from db.schemas import CommentMetadata
+from sqlalchemy import func
+from sqlalchemy import select
+from sqlalchemy.orm import defer
+from sqlalchemy.orm import load_only
+
+# May need rewrite after testing
```

Comment:
small comment here left in too

Reply:
Not commented by me so unsure whether to remove